test/utils/dom/test_scrollable.ts

```
@@ -0,0 +1,103 @@
+interface MockElement {
+  scrollHeight: number;
+  clientHeight: number;
+  overflowY?: string;
+  overflow?: string;
+}
+
+// Saved before the first stub so each test's afterEach can put the
+// real implementation (or undefined) back. Without this the stub
+// would leak into any subsequent test file run in the same process.
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+let originalGetComputedStyle: any =
```

ESLint

[error] 'originalGetComputedStyle' is never reassigned. Use 'const' instead.

(prefer-const)